packages/apollo-utils/src/sentry/apolloSentryPlugin.ts

```diff
@@ -81,32 +83,38 @@ export const sentryPlugin: ApolloServerPlugin<BaseContext> = {
             stack: err.stack, // parity with Sentry setup
           });
 
-          Sentry.withScope((scope) => {
-            // kind of operation == query/mutation/subscription
-            scope.setTag('kind', operationKind);
-            scope.setExtra('query', operationQuery);
-            scope.setExtra('variables', operationVariablesJson);
+          const scope = Sentry.getCurrentScope();
+          // kind of operation == query/mutation/subscription
+          scope.setTag('kind', operationKind);
+          scope.setExtra('query', operationQuery);
+          scope.setExtra('variables', operationVariablesJson);
 
-            if (requestId !== undefined) {
-              scope.setTag('graphRequestId', requestId);
-            }
-            if (requestTraceId !== undefined) {
-              scope.setTag('traceId', requestTraceId);
-            }
-            if (requestUserId !== undefined) {
-              scope.setTag('userId', requestUserId);
-            }
-            if (err.path) {
-              scope.addBreadcrumb({
-                category: 'query-path',
-                message: err.path.join(' > '),
-                level: 'debug',
-              });
-            }
+          if (requestId !== undefined) {
+            scope.setTag('graphRequestId', requestId);
+          }
+          if (requestTraceId !== undefined) {
+            scope.setTag('traceId', requestTraceId);
+          }
+
+          if (err.path) {
+            scope.addBreadcrumb({
+              category: 'query-path',
+              message: err.path.join(' > '),
+              level: 'debug',
+            });
+          }
 
-            // report error
-            Sentry.captureException(err);
+          if (apiId !== undefined) {
+            scope.setTag('pocket-api-id', apiId);
+          }
+
+          scope.setUser({
+            id: (requestEncodedId as string) || undefined,
+            ip_address: (gatewayIpAddress as string) || undefined,
```

Comment:
we added the x-forwarded-for final value as a header, I think called originIp? as an alternative

Reply:
@kschelonka is that in the router? I was using the docs on https://studio.apollographql.com/graph/pocket-client-api/variant/current/home